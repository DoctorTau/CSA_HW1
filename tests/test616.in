435
-4468 -1849 8980 3998 1116 -5572 5771 4327 -7322 3758 6867 -9261 7944 7242 -1433 1165 4831 4240 -4601 9390 -576 -584 9606 -1186 -3628 7654 -9065 4781 -4407 -5275 6948 6789 2413 -6017 -3280 -9748 -8255 2360 -1634 6501 -4285 8083 -4855 5312 -4287 -5243 3902 4437 -3873 6640 -9739 2293 -2609 -9478 2092 7807 2613 -5415 -5201 -6799 4778 3359 -609 6846 -9695 3856 106 -7549 -603 5524 1646 6424 -1865 1713 6412 -8640 8401 -3447 961 7861 -6646 7265 -8959 -6200 4465 6036 -4780 5806 3483 7911 -3213 -7349 -4109 3705 6819 1465 7651 4208 -4553 -5666 9595 2839 3620 2325 -4238 -7469 3054 9905 -3626 -3355 8191 3312 -1713 4939 -1107 -2386 -7771 3974 8463 -4135 -994 4754 -4309 -1012 -3460 5057 3522 5297 -4851 -8874 5350 -9867 3234 5506 -6440 2562 -1205 6617 2253 5124 -8067 -4531 1626 -8598 -5670 2244 3230 7436 -2212 7189 -3743 7027 810 -557 -1456 5909 2939 114 -6361 399 -6841 9932 1107 -4490 3967 6549 5852 512 -9471 -5466 2204 8234 -6442 4271 -2483 -8892 -7843 -9685 -5511 -9748 -3230 5527 8831 -3716 7117 1498 2256 -2121 -5665 7687 -7369 4433 -8264 -6801 4235 7652 -4734 -9932 7230 -8273 -9697 7514 2198 5282 -2853 -9446 1727 -6393 9763 -30 -3056 4479 6543 -6398 1722 5166 1685 8535 7158 -436 -2400 -187 1825 -4565 -5712 6374 9716 2981 4560 5547 7111 9572 -3932 3367 6484 -172 4635 7509 -4399 3205 -5549 4505 -6894 -2224 5062 -6011 -7545 3899 4427 -3025 4655 3305 -6446 -7343 -2284 1004 2232 7956 8491 9074 -6936 8113 7697 -9178 -332 5511 1893 6052 4819 8420 9145 1914 9681 9284 1705 -8918 -7973 -2596 -5872 -7636 6118 -824 115 2859 -2239 7766 -273 -4472 9993 -822 -5626 -8464 -7180 2684 6341 434 9855 -8736 4027 -5718 1661 -1320 8993 3050 3763 -1589 -8808 -9405 5653 -5507 -3455 621 -7311 9688 7971 6378 -5917 -9355 -8839 -3960 -8860 -8673 -8874 7379 -7292 4616 -9402 5436 -6906 8182 -2788 -7912 -6833 -889 1040 2562 -3336 4264 -2563 -4535 -207 4410 -7104 2963 1629 3021 6741 4393 -567 1923 -5384 -2378 605 5563 758 523 -1934 -7908 9488 -5391 -7388 -2138 1065 -7083 7568 -1490 6952 4732 2896 -428 -4824 8327 -9783 8775 -7911 -3127 5666 3270 2936 -9180 -3821 -6766 -1207 498 6840 5286 4130 -998 -4246 -5807 3967 -9543 -1609 -8927 -9163 -7636 -5111 -1647 -6674 2776 6429 5927 8189 3421 -1350 1224 -906 -8411 2422 -5593 -7855 -6236 -1735 4933 7915 -7927 -5090 4228 -2333 120 -7107 5557 4134 2468 -3283 -6750 -6765 5922 -8503 8122 -3163 -9177 5832 902 -3933 