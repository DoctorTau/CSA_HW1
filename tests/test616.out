-4468 -5572 -7322 -576 -584 -1186 -3628 -3280 -9748 -1634 -9478 -8640 -6646 -6200 -4780 -5666 -4238 -3626 -2386 -994 -1012 -3460 -8874 -6440 -8598 -5670 -2212 -1456 -4490 -5466 -6442 -8892 -9748 -3230 -3716 -8264 -4734 -9932 -9446 -30 -3056 -6398 -436 -2400 -5712 -3932 -172 -6894 -2224 -6446 -2284 -6936 -9178 -332 -8918 -2596 -5872 -7636 -824 -4472 -822 -5626 -8464 -7180 -8736 -5718 -1320 -8808 -3960 -8860 -8874 -7292 -9402 -6906 -2788 -7912 -3336 -7104 -5384 -2378 -1934 -7908 -7388 -2138 -1490 -428 -4824 -9180 -6766 -998 -4246 -7636 -6674 -1350 -906 -6236 -5090 -6750 